.gitignore

```diff
@@ -25,6 +25,7 @@ dockercfg
 packages/.vscode/settings.json
 **/testRequests/*.http
 *:Zone.Identifier
+.gradle
```

Comment:
Is this intentional?

Reply:
Yeah. I did a first time build of gradle from the root directory using: `yarn workspace @tupaia/meditrak-app run-android` and it installed it there.

I figured it's fine to have this here and it'll keep things clean if others do what I did 👍